action.yml

```
@@ -15,6 +15,11 @@ inputs:
     type: string
     required: false
     default: "https://dashboard.useparagon.com"
+  path:
+    type: string
+    description: 'Specify base path to Paragraph project, relative to the working directory.'
+    required: false
+    default: "./"
```

```suggestion
  subfolder:
    type: string
    required: false
    default: '.'
```

should match `paragraph-pull`